app/src/main/java/ch/epfl/skysync/models/flight/Flight.kt

```diff
@@ -12,5 +12,5 @@ interface Flight {
   val date: LocalDate
   val timeSlot: TimeSlot
   val vehicles: List<Vehicle>
-  val id: String
+  val id: String?
```

Comment:
Why can the id of a flight be null ? Don't forget to ensure that appropriate null checks are added for code use and tests as well

Reply:
This id is the id that we will get from firebase and with my knowledge about it. I don't think we can get it before writing it in the database. But as we do not have implemented the model view for this part yet so I am not sure about it. It was also easier for me as I did not have to generate random ids for testing it.